### PR TITLE
Yuval/comments on patch21

### PR DESCRIPTION
**Pull request by yuval-lb**

### CODE REVIEW

File: cinder/volume/drivers/lightos.py [hunk 1]
```diff
@@ -408,7 +408,7 @@ def create_cloned_volume(self, volume, src_vref):
             self._create_snapshot(project_name,
                                   snapshot_name, src_volume_name)
         except Exception as e:
-            LOG.warn(
+            LOG.warning(
```
Comment (ronen-lb) [inline cinder/volume/drivers/lightos.py:411]: +1

File: cinder/volume/drivers/lightos.py [hunk 2]
```diff
@@ -799,8 +799,6 @@ def delete_volume(self, volume):
             msg = ('Failed to delete LightOS volume with UUID'
                    ' %(uuid)s project %(project_name)s' % (
                        dict(uuid=lightos_uuid, project_name=project_name)))
-
-            msg = _(msg)
             raise exception.VolumeBackendAPIException(message=msg)
```
Comment (ronen-lb) [inline cinder/volume/drivers/lightos.py:802]: why was this dropped, you recast msg generated in line 799 so isnt this needed?

File: cinder/volume/drivers/lightos.py [hunk 3]
```diff
@@ -799,8 +799,6 @@ def delete_volume(self, volume):
             msg = ('Failed to delete LightOS volume with UUID'
                    ' %(uuid)s project %(project_name)s' % (
                        dict(uuid=lightos_uuid, project_name=project_name)))
-
-            msg = _(msg)
```
Comment (ronen-lb) [inline cinder/volume/drivers/lightos.py:803]: looks fine, though would be happy to be understand/review which functions 
are invoked by operator and require the original change and which dont

**Review by ronen-lb — APPROVED** [review]

**Comment by muliby-lb** [comment]
@yuval-lb is this still needed? Merge/close?

**Pull request closed without merging**